srcpkgs/dmd2.091/template

```diff
@@ -0,0 +1,78 @@
+# Template file for 'dmd2.091'
+pkgname=dmd2.091
+version=2.091.0
+revision=1
+create_wrksrc=yes
+hostmakedepends="which"
+makedepends="dmd2.081 git"
+depends="gcc"
+short_desc="Digital Mars D compiler, 2.091"
+maintainer="Auri <me@aurieh.me>"
+license="BSL-1.0"
+homepage="http://www.digitalmars.com/d/2.0/"
+distfiles="
+ https://github.com/dlang/dmd/archive/v${version}.tar.gz>dmd-${version}.tar.gz
+ https://github.com/dlang/druntime/archive/v${version}.tar.gz>druntime-${version}.tar.gz
+ https://github.com/dlang/phobos/archive/v${version}.tar.gz>phobos-${version}.tar.gz
+ http://downloads.dlang.org/releases/2.x/${version}/dmd.${version}.linux.tar.xz"
+checksum="bcca38f4b80b51ae0a1955dc29e6cbfaa4d01d94869ddfcacf9292898c34343a
+ cc8ba196b08cca04488d97d0e35c878f7351b7d8128aec90eaa312d2bd9a6af2
+ df9f81eecda4366adc5ca9ed31a0b4474c248544b120ff6fa3f6cb45917d8769
+ 6e4bd4ee73a553921cdca7e3e21768b842b18186d07c0b4898fbd13a359875c0"
+conf_files="/etc/dmd.conf"
+provides="d-compiler-${version}_${revision}"
+conflicts="dmd-bootstrap dmd2.081 libphobos2.081"
+nopie=yes
+LDFLAGS="-lpthread"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*) _archbits=64;;
+	i686) _archbits=32;;
+	*) broken="unsupported arch upstream";;
+esac
+
+post_extract() {
+	rm -rf dmd druntime phobos
```

Comment:
Could this be `rmdir`?

Reply:
I'm not experienced enough with `xbps-src` to tell. Are the directories supposed to be empty, even after a failed build?

Reply:
If they are empty directories, it should be ok.

Reply:
Interesting, apparently the dirs don't even exist after `do_extract`. I wonder why that `rm` is in the original template.